analysis/crystal/crystal.go

```diff
@@ -0,0 +1,85 @@
+package crystal
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+// Host is the base URL for the analysseur API.
```

Comment:
Maybe we should rename these to ruby-analyzer.exercism.io and crystal-analyzer.exercism.io or something.